src/bot.ts

```diff
@@ -139,18 +146,24 @@ dfusionService.watchOrderPlacement({
     datesDescription += `  - *Expires*: \`${moment(validUntil).calendar()} GMT\`, \`${moment(validUntil).fromNow()}\``
 
     // Format the amounts
-    const sellAmountFmt = sellToken
-      ? formatAmount(new BN(priceDenominator.toString()), sellToken.decimals)
-      : priceDenominator
-    const buyAmountFmt = buyToken ? formatAmount(new BN(priceNumerator.toString()), buyToken.decimals) : priceNumerator
+    // TODO: Allow to use BN, string or BigNumber or all three in the format. Review in dex-js
+    const sellAmountFmt = formatAmount(new BN(priceDenominator.toString()), sellToken.decimals)
+    const buyAmountFmt = formatAmount(new BN(priceNumerator.toString()), buyToken.decimals)
+    const fillSellAmountFmt = formatAmountFull(
+      new BN(priceNumerator.multipliedBy(FACTOR_TO_FILL_ORDER).toString()),
+      buyToken.decimals
+    )
+    const buyAmountFullFmt = formatAmountFull(new BN(priceDenominator.toString()), sellToken.decimals)
 
     // Compose message using markdown
     // TODO: Provide the link to the front end: https://github.com/gnosis/dex-telegram/issues/3
     // TODO: Should we publish even if the user doesn't have balance. Should we include the balance of the user? he can change it...
     const message = `Sell *${sellAmountFmt}* \`${sellTokenLabel}\` for *${buyAmountFmt}* \`${buyTokenLabel}\`
 
   - *Price*:  1 \`${sellTokenLabel}\` = ${price} \`${buyTokenLabel}\`
-${datesDescription}`
+${datesDescription}
+
+Fill the order here: ${WEB_BASE_URL}/trade/${buyTokenLabel}-${sellTokenLabel}?sell=${fillSellAmountFmt}&buy=${buyAmountFullFmt}` // TODO:
```

Comment:
Suggestion for a new utils function: `function buildTradeUrl(...)`

Reply:
issue: https://github.com/gnosis/dex-js/issues/9